testnet/stacks-node/src/neon_node.rs

```diff
@@ -2007,6 +2039,41 @@ impl InitializedNeonNode {
             sunset_burn,
             burn_block.block_height,
         );
+
+        let cur_burn_chain_tip = SortitionDB::get_canonical_burn_chain_tip(burn_db.conn())
+            .expect("FATAL: failed to query sortition DB for canonical burn chain tip");
+
+        // last chance -- confirm that the stacks tip and burnchain tip are unchanged (since it could have taken long
+        // enough to build this block that another block could have arrived).
+        if let Some(stacks_tip) = chain_state
+            .get_stacks_chain_tip(burn_db)
+            .expect("FATAL: could not query chain tip")
+        {
+            if stacks_tip.anchored_block_hash != anchored_block.header.parent_block
+                || parent_consensus_hash != stacks_tip.consensus_hash
+                || cur_burn_chain_tip.sortition_id != burn_block.sortition_id
+            {
+                debug!(
+                    "Cancel block-commit for block {} tx-count {} height {} off of {}/{} with microblock parent {} (seq {}) in burn block {} ({}) sortition {}; attempt {}.  New tip is {}/{}, sortition {}",
+                    &anchored_block.block_hash(),
+                    anchored_block.txs.len(),
+                    anchored_block.header.total_work.work,
+                    &parent_consensus_hash,
+                    &anchored_block.header.parent_block,
+                    &anchored_block.header.parent_microblock,
+                    &anchored_block.header.parent_microblock_sequence,
+                    &burn_block.burn_header_hash,
+                    burn_block.block_height,
+                    burn_block.sortition_id,
+                    attempt,
+                    stacks_tip.anchored_block_hash,
+                    stacks_tip.consensus_hash,
+                    cur_burn_chain_tip.sortition_id
```

Comment:
This should be a k-v structured log. Message should just be "Canceling block-commit for block", and then all of the rest of these values should be k-v pairs.

Reply:
Fixed